bases/next.json

```diff
@@ -11,7 +11,7 @@
     "noEmit": true,
     "esModuleInterop": true,
     "module": "esnext",
-    "moduleResolution": "node",
+    "moduleResolution": "bundler",
```

Comment:
This would be a breaking change in Next.js since its only TS 5+

While we're at it, we could probably remove `target` since Next.js doesn't use it.